project/Build.scala

```diff
@@ -55,7 +55,7 @@ object MyScalaJSPlugin extends AutoPlugin {
 object Build {
   val referenceVersion = "0.27.0-RC1"
 
-  val baseVersion = "0.28.0"
+  val baseVersion = "3.0.0"
```

Comment:
Shouldn't this be `3.0.0-M1` ?

Reply:
In the case of RC versions, we only set it to x.y.z-RC1 on release. Otherwise, it is always x.y.z. So I assume the same pattern holds for milestone versions.

Reply:
I think we could have used an RC1 suffix before release but it didn't matter much. This is different because the `scalaBinaryVersion` (and therefore artefact suffix) of `3.0.0-M1-bin-SNAPSHOT` is `3.0.0-M1`, whereas for `3.0.0-bin-SNAPSHOT` it will be `3.0.0`, (contrast with `0.27.0-bin-SNAPSHOT` and `0.27.0-RC1-bin-SNAPSHOT` which both have binary version `0.27`)

Reply:
In the SBT plugin we have:

```scala
      scalaBinaryVersion := {
        if (isDotty.value)
          scalaVersion.value.split("\\.").take(2).mkString(".")
        else
          scalaBinaryVersion.value
      },
```

Does that mean the `if` condition should be modified to exclude the versions that start with `3`?

Reply:
Yes, hopefully the default implementation of scalaBinaryVersion works fine for version numbers >= 3